cfme/infrastructure/pxe.py

```diff
@@ -143,9 +144,10 @@ class PXEServer(Updateable, Pretty, sentaku.Element):
     """
     pretty_attrs = ['name', 'uri', 'access_url']
 
-    def __init__(self, name=None, depot_type=None, uri=None, userid=None, password=None,
+    def __init__(self, parent, name=None, depot_type=None, uri=None, userid=None, password=None,
```

Comment:
the parent is the sentaku app in that case, naming it `sentaku_app` there might be more to the point